spec/unit/type/group_spec.rb

```diff
@@ -6,8 +6,16 @@
     @class = Puppet::Type.type(:group)
   end
 
-  it "should have a system_groups feature" do
-    expect(@class.provider_feature(:system_groups)).not_to be_nil
+  it 'should default to `present`' do
+    expect(@class.new(:name => "foo")[:ensure]).to eq(:present)
+  end
+
+  it 'should default to `present`' do
+    puts "farts"
```

Comment:
hmm, is this something you were using for debugging and forgot to remove?

Reply:
oh wow.. yes... sorry about that